lib/hammer_cli/utils.rb

```diff
@@ -8,10 +8,15 @@ def format(params)
         name = name[0]
         params[name.to_s] || params[name.to_sym]
       end
-
-      self.gsub(/%[<]([^>]*)[>]/, '%').gsub(/%[{]([^}]*)[}]/, '%s') % array_params
+      self.gsub(/%[<]([^>]*)[>]/, '%')
+          .gsub(/%[{]([^}]*)[}]/, '%s')
+          .gsub(/(\%\W?[^bBdiouxXeEfgGaAcps])/) do |match|
+            match.gsub(/\%/, '%%')
+          end % array_params
```

Comment:
@tstrachota , 
Yes, corrected code as per above regex expression & updated the PR. 
Thank you.